cypress/integration/materialUIViewer.js

```diff
@@ -0,0 +1,89 @@
+describe('The Material-UI Viewer', () => {
+  // Load the viewer
+  before(() => {
+    cy.visit('/')
+    cy.wait(10000)
+    cy.waitForReact(10000, '.uiContainer')
+
+  })
+
+  beforeEach(() => {
```

Comment:
Not positive why we need to do this but for some reason when we run in headless mode when the data loads the drawer is collapsed causing any remaining tests to fail. Using `beforeEach` for now because the timing of this behavior can change each time. See this [video](https://drive.google.com/file/d/19XREh3zGxlU4Nlvdcs5MQERCMGgR_qMM/view?usp=sharing) generated by Cypress to see this happening.

Reply:
Yes, odd. Lovely, though, that we have the video to understand what is happening.